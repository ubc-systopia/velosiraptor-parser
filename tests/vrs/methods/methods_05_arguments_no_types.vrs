segment MyUnit {
    // method with pre-condition
    fn foo(bar) -> bool
    {

    }
}