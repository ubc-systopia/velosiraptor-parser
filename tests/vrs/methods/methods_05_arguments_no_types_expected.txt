error: expected `:`, found `)`
     --> tests/vrs/methods/methods_05_arguments_no_types.vrs:3:15
      |
    3 |      fn foo(bar) -> bool
      |                ^ help: add `:` here.
